api/mapping/forms.py

```diff
@@ -4,23 +4,24 @@
 from django.contrib.auth.forms import UserCreationForm
 from django.contrib.auth.models import User
 from django.core.exceptions import ValidationError
-from django.forms.models import ModelChoiceField
+from django.forms.models import ModelChoiceField, ModelForm
 
 from mapping.models import (OPERATION_CHOICES, DataPartner, Document,
-                            DocumentFile, DocumentType, OmopField, OmopTable,
+                            DocumentFile, DocumentType, FLAG_CHOICES, OmopField, OmopTable,
                             ScanReport, VOCABULARY_CHOICES)
 from xlsx2csv import Xlsx2csv
 
 class MyModelChoiceField(ModelChoiceField):
```

Comment:
This is because we changed the model return to self.id, so the selection boxes where just showing numbers instead of names of data provides or document types etc. It is just used instead of a ModelChoiceField to return the object name instead of the id for the selection box. I can rename it if you have a better name suggestion